src/openforms/forms/tests/test_migrations.py

```diff
@@ -435,3 +435,57 @@ def test_fix_badly_formatted_rules(self):
             self.rule_badly_formatted.json_logic_trigger,
             {"==": [{"var": "selectBoxes.a"}, True]},
         )
+
+
+class TestLogicParsing(TestMigrations):
+    migrate_from = "0002_auto_20210917_1114_squashed_0045_remove_formstep_optional"
+    migrate_to = "0046_convert_advanced_logic"
+    app = "forms"
+
+    def setUpBeforeMigration(self, apps):
```

Comment:
you haven't set up a form that uses this form definition, so the migration path that creates `FormLogic` records will never be hit in this test.

Please set up a form with a form step that uses this form definition you create.